internal/server/server.go

```diff
@@ -13,7 +13,7 @@ import (
 
 //go:generate sh -c "protoc -I../../vendor/proto/api-common-protos -I ../.. ../../internal/server/proto/server.proto --go_out=plugins=grpc:../.. --go-json_out=../.."
 //go:generate mv ./proto/server.pb.json.go ./gen
-//go:generate mockery -all -case underscore -dir ./gen -output ./gen/mocks
+//go:generate mockery --all --case underscore --dir ./gen --output ./gen/mocks
```

Comment:
This appears unrelated to this PR. Please remove.

Reply:
Will do :)

Reply:
I believe Mockery version 2.x+ uses double hyphens but we're on an older version so we use single.

Reply:
But does the nix environment use a newer one?  Because in my nix shell, the default (single hypen) wasn't working IIRC

Reply:
I'm not sure, I don't use it 😄 But when I was first getting started I just installed the latest Mockery and this was an issue I ran into, rolling back to `1.1.2` fixed it for me.

https://github.com/hashicorp/waypoint/blob/7d6229c3167209b80459e67bdc669c32465aceb0/tools/tools.go#L22-L23

Reply:
Looks like it should technically be installing the right version.... https://github.com/hashicorp/waypoint/blob/7d6229c3167209b80459e67bdc669c32465aceb0/shell.nix#L61-L62

Reply:
Really, really strange. I'll have to double check my environment since shell.nix reports the 1.x version too 😮

Reply:
Reverted the change, sorry!